lib/svg-sprite/mode/standalone.js

```diff
@@ -64,42 +64,44 @@ SVGSpriteStandalone.prototype._layout = function(files, cb, extend) {
     let doctypeDeclaration = null;
     for (const [index, shape] of this._spriter._shapes.entries()) {
         // Skip non-master shapes
-        if (!shape.master) {
-            xmlDeclaration = xmlDeclaration || shape.xmlDeclaration;
-            doctypeDeclaration = doctypeDeclaration || shape.doctypeDeclaration;
-            this.data.shapes[index] = {
-                ...this.data.shapes[index],
-                selector: {
-                    dimensions: shape.state ? [{
-                        expression: `${format(this.config.dimensions, shape.base)}:${shape.state}`,
-                        raw: `${format(this.config.dimensions, shape.base)}:${shape.state}`,
-                        first: true,
-                        last: false
-                    }, {
-                        expression: format(this.config.dimensions, `${shape.base}\\:${shape.state}`),
-                        raw: format(this.config.dimensions, `${shape.base}:${shape.state}`),
-                        first: false,
-                        last: true
-                    }] : [{
-                        expression: format(this.config.dimensions, shape.base),
-                        raw: format(this.config.dimensions, shape.base),
-                        first: true,
-                        last: true
-                    }]
-                }
-            };
+        if (shape.master) {
+            continue;
+        }
 
-            Object.defineProperty(this.data.shapes[index], '_svg', {
-                enumerable: false,
-                writable: true
-            });
+        xmlDeclaration = xmlDeclaration || shape.xmlDeclaration;
+        doctypeDeclaration = doctypeDeclaration || shape.doctypeDeclaration;
+        this.data.shapes[index] = {
+            ...this.data.shapes[index],
+            selector: {
+                dimensions: shape.state ? [{
+                    expression: `${format(this.config.dimensions, shape.base)}:${shape.state}`,
+                    raw: `${format(this.config.dimensions, shape.base)}:${shape.state}`,
+                    first: true,
+                    last: false
+                }, {
+                    expression: format(this.config.dimensions, `${shape.base}\\:${shape.state}`),
+                    raw: format(this.config.dimensions, `${shape.base}:${shape.state}`),
+                    first: false,
+                    last: true
+                }] : [{
+                    expression: format(this.config.dimensions, shape.base),
+                    raw: format(this.config.dimensions, shape.base),
+                    first: true,
+                    last: true
+                }]
+            }
+        };
 
-            this.data.shapes[index].__defineSetter__('svg', function(svg) {
-                this._svg = svg;
-            });
+        Object.defineProperty(this.data.shapes[index], '_svg', {
```

Comment:
Maybe this changes should be in another PR?

Reply:
I didn't change that, make sure you have whitespace disabled